module-2/backend/week_two.md

```diff
@@ -26,16 +34,37 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+*Team.find(4).owner*
+
 3. What do they allow you to do?
+
+*Anything I want, for the most part.*
+
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+*The student table and the teacher table connect to a join table via their respective IDs.*
+
 8. Define foreign key, primary key, and schema.
+
+*A foreign key is the primary key ID of another table in a database. A primary key is used to identify specific instances in a database. A schema is a representation of the structure of a database that shows the attributes of each table and any relationships between tables*
+
 9. Describe the relationship between a foreign key on one table and a primary key on another table.
-10. What are the parts of an HTTP response?
+
+*If a horse has one jockey but a jockey can have many horses, the horses table would have a place for jockey_id.*
+
+10. What are the parts of an HTTP response? 
+
+*Body, headers, status code, verb*
```

Comment:
Don't need the verb on the response. That's part of the request. Otherwise looks good!